Lib/idlelib/calltips.py

```diff
@@ -139,21 +139,32 @@ def get_argspec(ob):
         ob_call = ob.__call__
     except BaseException:
         return argspec
-    if isinstance(ob, type):
-        fob = ob.__init__
-    elif isinstance(ob_call, types.MethodType):
```

Comment:
Why these removals? This should be justified by discussion/examples on the issue first.